runtime/engine/src/test/java/io/aklivity/zilla/runtime/engine/test/internal/binding/config/TestBindingOptionsConfig.java

```diff
@@ -15,13 +15,17 @@
  */
 package io.aklivity.zilla.runtime.engine.test.internal.binding.config;
 
+import java.util.List;
 import java.util.function.Function;
 
 import io.aklivity.zilla.runtime.engine.config.OptionsConfig;
 
 public final class TestBindingOptionsConfig extends OptionsConfig
 {
     public final String mode;
+    public final List<String> catalogs;
+    public final List<Guard> guards;
```

Comment:
Let's change `guards` of type `List<Guard>` to `authorization` (one, not list) of type `TestAuthorizationConfig` with a `String name` (of the referenced guard) and `credentials` (the token). This is more or less how we do it for HTTP binding with `HttpAuthorizationConfig` except that one is extracting the token from a patterned string, and here we can just specify the token directly instead.

The syntax in `yaml` should also follow the same approach as other binding `authorization` syntax.

```
bindings:
  test0:
    type: test
    kind: server
    options:
        authorization:
            jwt0:
                credentials: token
```